tools/tests/test_lint.py

```
@@ -1144,3 +1456,313 @@ def test_non_mapping_dispositions_and_staffing_are_findings(tmp_path):
+    # body stays quiet here.
+    assert len(charter.read_text(encoding="utf-8")) > lint.CHARTER_BUDGET_CHARS
+    assert [f for f in lint.run(tmp_path) if "doctrine-budget" in f] == []
+
+
+def _set_description(root: Path, value: str) -> None:
+    """Rewrite the charter cell's description, leaving everything else alone."""
+    cell = root / "skills" / "charter" / "SKILL.md"
+    lines = cell.read_text(encoding="utf-8").splitlines()
+    close = lines.index("---", 1)
+    kept = [ln for ln in lines[1:close] if not ln.startswith("description:")]
+    rebuilt = ["---"] + kept + ["description: " + value] + lines[close:]
```

Ruff (0.16.2)

[warning] Consider iterable unpacking instead of concatenation

Replace with iterable unpacking

(RUF005)